shared/utils.ts

```diff
@@ -1,3 +1,13 @@
+import _ from 'lodash';
+
+export const capitalizeStringIfUppercase = (value: string) => {
+	if (value.toUpperCase() === value) {
+		return _.upperFirst();
```

Comment:
nit: You need to pass the value to the function😉

Reply:
hahah, that's indeed beneficial. thanks for rechecking. good that all my tests capture it 😆